smartlisten.lic

```diff
@@ -10,23 +10,25 @@ no_kill_all
 
 listen_skills = get_settings.listen_skills
 
+listen_skills += ['Life Magic', 'Elemental Magic', 'Lunar Magic', 'Holy Magic'] unless DRStats.barbarian? || DRStats.thief?
+
 if DRStats.barbarian?
   listen_skills += ['Expertise', 'Inner Fire']
-  listen_skills -= ['Arcana', 'Life Magic', 'Elemental Magic', 'Lunar Magic', 'Holy Magic']
+  listen_skills -= ['Arcana']
 elsif DRStats.bard?
-  listen_skills += ['Bardic Lore', 'Elemental Magic']
+  listen_skills += ['Bardic Lore']
 elsif DRStats.cleric?
-  listen_skills += ['Theurgy', 'Holy Magic']
+  listen_skills += ['Theurgy']
 elsif DRStats.empath?
-  listen_skills += ['Empathy', 'Life Magic']
+  listen_skills += ['Empathy']
 elsif DRStats.moon_mage?
-  listen_skills += ['Astrology', 'Lunar Magic']
+  listen_skills += ['Astrology']
 elsif DRStats.necromancer?
   listen_skills += ['Thanatology', 'Arcane Magic']
 elsif DRStats.paladin?
-  listen_skills += ['Endurance', 'Holy Magic']
+  listen_skills += ['Endurance']
 elsif DRStats.ranger?
-  listen_skills += ['Scouting', 'Life Magic']
+  listen_skills += ['Scouting']
 elsif DRStats.thief?
   listen_skills += ['Backstab', 'Inner Magic']
```

Comment:
❓ Is this still called Backstab? I thought it was renamed

Reply:
sure is, its referred to in game as blindside but the skill is backstab still